src/assets/stylesheets/DownloadPanel.scss

```diff
@@ -0,0 +1,37 @@
+@use "./rpf_design_system/spacing" as *;
+@use "./rpf_design_system/colours.scss" as *;
+@use "./rpf_design_system/font-weight" as *;
+@use "./rpf_design_system/font-size.scss" as *;
+
+.download-panel__content {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: $space-0-5;
+  align-self: stretch;
+}
+
+.download-panel__subtitle {
+  @include font-size-1(regular);
+  font-weight: $font-weight-bold;
+}
+
+.download-panel__button-container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-self: stretch;
+  gap: $space-0-75;
+  align-items: stretch;
+  padding-bottom: $space-1;
+  border-bottom: 1px solid $rpf-grey-150;
+}
+
+.download-panel__button {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 0;
```

Comment:
I'm kinda surprised all this flex stuff is needed if you're applying the styles from the editor button

Reply:
Yes I've played around with it and removed what wasn't having an effect - thanks